src/rard/research/views/mention.py

```diff
@@ -1,9 +1,10 @@
 # from django.contrib.postgres.search import SearchQuery, SearchRank, \
 #     SearchVector
-from itertools import chain
 
+from django.apps import apps
 from django.contrib.auth.mixins import LoginRequiredMixin
-from django.db.models import Q
+from django.db.models import CharField, F, Q, Value
```

Comment:
Import StringAgg (see suggestion below)
```suggestion
from django.db.models import CharField, F, Q, Value
from django.contrib.postgres.aggregates import StringAgg
```